src/Corax/Constants.cs

```diff
@@ -6,6 +6,9 @@ namespace Corax
 {
     public static class Constants
     {
+        public const string BeforeAllKeys = "BeforeAllKeys-a8e5f221-613e-4eae-9962-2689e7c44506";
+        public const string AfterAllKeys = "AfterAllKeys-3622a0bb-1cf4-4200-b830-5e937d57ac99";
```

Comment:
Debug leftovers?

Reply:
No, they are being used, and I wanted to ensure that we are using values that no one else ever would